mcs/class/Makefile

```diff
@@ -300,6 +298,11 @@ xbuild_4_0_dirs := \
 	$(xbuild_2_0_dirs)	\
 	Microsoft.Build
 
+ifdef HAVE_BTLS
```

Comment:
Ah, I knew there was some better way but couldn't find it. This doesn't work for the btls executable though right?